src/ar/mangaswat/src/eu/kanade/tachiyomi/extension/ar/mangaswat/MangaSwat.kt

```diff
@@ -39,6 +41,14 @@ class MangaSwat :
 
     private val preferences by getPreferencesLazy()
 
+    private val apiBaseUrl = "https://appswat.com/v2/api/v2"
+
+    private val apiDateFormat by lazy {
+        SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US).apply {
```

Comment:
The pattern not contains any locale string
```suggestion
        SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.ROOT).apply {
```